test/common/child_process.js

```diff
@@ -1,6 +1,7 @@
 'use strict';
 
 const assert = require('assert');
+const { spawnSync } = require('child_process');
```

Comment:
Since `child_process` is required unconditionally, we might as well get `execFileSync` here and remove conditional import in https://github.com/nodejs/node/blob/67487fe6265a34d26e8a9cd9e4d2e85dfb2eeb31/test/common/child_process.js#L18-L19

Reply:
The behavior of `execFileSync` is not quite desirable for tests that expect the command to fail and needs to check how it fails, it also hides stderr which is commonly checked by these tests.

Reply:
oh wait, you meant just merging the `require` down below, that makes much more sense :)